api/v1alpha1/shipwrightbuild_types.go

```diff
@@ -44,3 +47,13 @@ type ShipwrightBuildList struct {
 func init() {
 	SchemeBuilder.Register(&ShipwrightBuild{}, &ShipwrightBuildList{})
 }
+
+// IsReady returns true if all of the Ready condition has true status.
+func (status ShipwrightBuildStatus) IsReady() bool {
+	for _, condition := range status.Conditions {
+		if condition.Type == "Ready" && condition.Status == metav1.ConditionTrue {
+			return true
+		}
+	}
+	return false
+}
```

Comment:
The comment says "all of the Ready conditions have true status", but doesn't this method return `true` for the first condition it finds where `Type = "Ready"` and 'Status = true`? Seems like it should be flipped some.
```suggestion
// IsReady returns true if all of the Ready condition has true status.
func (status ShipwrightBuildStatus) IsReady() bool {
	for _, condition := range status.Conditions {
		if condition.Type == "Ready" && condition.Status != metav1.ConditionTrue {
			return false
		}
	}
	return true
}
```

Reply:
fixed